examples/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 set ( EXAMPLES
       simple_trixi_controller.c
-      simple_trixi_controller.f90 )
+      simple_trixi_controller.f90
+      trixi_controller_data.c
+      trixi_controller_data.f90 )
```

Comment:
What do you think? Should the data access be demonstrated in separate examples like those, or integrated into the existing ones?

Reply:
I think for now I'd leave it as it is until we figure out a different approach is better/more useful.